src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -269,6 +272,9 @@ final class Inflector
         // circuses (circus)
         ['suc', 3, true, true, 'cuses'],
 
+        // conspectuses (conspectus), prospectuses (prospectus)
+        ['sutcep', 6, true, true, 'pectuses'],
```

Comment:
if this line is removed, will the tests still pass?

Reply:
no, the tests failed
![image](https://user-images.githubusercontent.com/19323431/117654176-ed4d5600-b184-11eb-94af-e5f2e6489eb7.png)